Epic0/styles

Annet-Lovett opened the pull request.

Commit:
add styles for toolbar, titles, errors

Commit:
icon names
dimens size
colors and size in icons
delete icon back from toolbar

Commit:
statusBarColor

Commit:
dto

onecoffeeplz reviewed (changes requested):
не хватает некоторых ассетов: плейхолдер на отсутствующий логотип компании, иконка фильтра, картинка нет интернета (череп), иконка вперед (стрелка вправо), иконка крестик

Commit:
add:
ic_close_cross
ic_filter
ic_forward
ic_vacancy_placeholder
image_placeholder_no_internet

Commit:
color_checked_menu

nastyavorontsova reviewed (commented):
Посмотрела, хочу уточнить по инпуту. Учтено ли, что он в каждой теме разный? в светлой теме фон должнен Light Gray, а в темной Gray
в светлой теме цвет hint должен быть Gray, в темной White Universal
сейчас он как будто единый, как для светлой так и для темной темы

Commit:
colors and dimens
navigationBarColor
windowLightNavigationBar

Commit:
textFontWeight

Commit:
di
